### PR TITLE
Fix build system when using CMake 3.12

### PR DESCRIPTION
**Pull request by Rombur**
I updated the cuda tester and all hell broke loose. @masterleinad can you check that this work on your machine.

I have disabled the cuda tester for now because without this patch I can compile anything.

### CODE REVIEW

File: cmake/configure/configure_1_cuda.cmake [hunk 1]
```diff
@@ -96,6 +102,10 @@ MACRO(FEATURE_CUDA_FIND_EXTERNAL var)
       SET(${var} FALSE)
     ENDIF()
 
+    # cuSOLVER requires OpenMP
+    FIND_PACKAGE(OpenMP)
+    SET(DEAL_II_LINKER_FLAGS "${DEAL_II_LINKER_FLAGS} ${OpenMP_CXX_FLAGS}")
```
Comment (tamiko) [inline cmake/configure/configure_1_cuda.cmake]: Acked - looks good. But I cannot test at the moment. (Living in IKEA boxes and no system with cuda available.)

Comment (masterleinad) [reply cmake/configure/configure_1_cuda.cmake]: Is the failing `cuda/solver_02` finally passing for you with this fix?

Comment (Rombur) [reply cmake/configure/configure_1_cuda.cmake]: No unfortunately :(

**Pull request merged**